sources/Outputs/src/ParDynModel.cpp

```diff
@@ -108,13 +109,19 @@ ParDynModel::writeSVCParameterSet(const inputs::SettingDataBase::Set& set, const
       if (genDefinition.hasTransformer())
         new_set->addParameter(helper::buildParameter("secondaryVoltageControl_XTfoPu_" + std::to_string(idx) + "_",
                                                      (genDefinition.isNetwork()) ? constants::generatorNucXPuValue : constants::generatorXPuValue));
+
+      if (genDefinition.q < genDefinition.qmax && genDefinition.q > genDefinition.qmin) {
```

Comment:
I'd align the codition as in the modelica model (check on volatge values also and <= >=)

Reply:
Regarding the voltages, it is not possible as initially URef=URegulated=targetV of the node

Reply:
I see, ok for me then! thx! we can still have <= and >=, I think

Reply:
never mind, it is correct as it is